CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+project("Libre Graph API" LANGUAGES CXX)
+
+cmake_minimum_required(VERSION 3.2)
+
+find_package(Qt5Core REQUIRED)
```

Comment:
No need to check for the packages at this level the add sub_directory should be sufficient.